engine/resources/lua/KeyboardInput.lua

```diff
@@ -2,13 +2,22 @@
 local Keyboard = {};
 Keyboard.key = {};
 for i = 0, 255, 1 do
-	Keyboard.key[i] = {false,false,false};
+	Keyboard.key[i] = {false, false, false};
 end
 
+
+Keyboard.key[347] = {false, false, false}; --mac cmd
```

Comment:
We need to figure out how to make this generic. 
The ```InputManager``` is supposed to handle this.